src/ray/raylet/node_manager.cc

```diff
@@ -496,8 +496,10 @@ void NodeManager::Heartbeat() {
 }
 
 void NodeManager::DoLocalGC() {
-  auto all_workers = worker_pool_.GetAllRegisteredWorkers();
-  for (const auto &driver : worker_pool_.GetAllRegisteredDrivers()) {
+  auto all_workers =
+      worker_pool_.GetAllRegisteredWorkers(false /* filter_dead_workers */);
```

Comment:
```suggestion
      worker_pool_.GetAllRegisteredWorkers(/* filter_dead_workers=*/false);
```
to stay consistent with other places in the code. same with below